packages/ui/src/theme/Color.ts

```diff
@@ -165,10 +165,488 @@ export enum Color {
   Yellow500 = '#B84807',
 }
 
+export enum ColorLightInternal {
```

Comment:
@MekhrangizY Can we add an extended comment at the start of the file with description of what these 4 enums represent and what needs to be changed if someone wants to update one of the colors?